### DISCUSSION TITLE
Fail to create journal entry in collection

### DISCUSSION THREAD

**Discussion by DickinsonSa** [TechbeeAT/jtxBoard#415] [General]
This is the scenario for this issue :

When I'm in the journals section of jtx and select the lower right button to create a new journal entry I'm given a blank form to insert the details, however  -

- the default collection visible isn't the one I want to create the entry in - so using the drop-down I select the collection I wish to use, I'm then thrown out of the new journal entry form back to the journals screen. Also... 

An empty journal entry has been created and, inside the collection that appeared before I tried to make the change in the drop-down (what I previously referred to as the default collection). 

This empty journal entry then needs to be deleted and the process restarted. This can be repeated again and again.

This can be frustrating especially if many details have been entered **before** the collection change is done. 

**Also**, when I delete the empty entry I'm thrown afterwards to the collections screen. The entry **is** deleted though.

**Comment by patrickunterwegs** [comment]
Hello @DickinsonSa , 
We had discussed the issue already about returning to the wrong page after deleting an entry here
https://github.com/TechbeeAT/jtxBoard/discussions/383

About creating new entries:
I have heard about this issue before but I couldn't reproduce it. 
Basically when changing the collection of an entry the app would:
- save the current entry
- create a copy of the entry in the new collection
- delete the current entry
- open the new copy

It seems like something is not done in the right order for you.
Which app and android versions are you using?

**Reply by DickinsonSa** [reply]
Yes, mine is a different order.
Android 12
v 2.3.0

**Reply by patrickunterwegs** [reply]
Hi @DickinsonSa , I will close this discussion now, that issue should hopefully be fixed in v2.4.0 through the issue #417
